### PR TITLE
use another bundle for example

### PR DESCRIPTION
**Pull request by OskarStark**

### CODE REVIEW

File: project/CONTRIBUTING.md [hunk 1]
```diff
@@ -551,7 +551,7 @@ but are encouraged to report release requests on the dedicated private Slack cha
 Releasing software is the last step to getting your bugfixes or new features to your user base,
 and should be done regularly, so that users are not tempted to use development branches.
 To know what you are going to release on branch 42.x, given that the last release on this branch is 42.3.1,
-go to `https://github.com/sonata-project/SonataCoreBundle/compare/42.3.1...42.x`.
+go to `https://github.com/sonata-project/SonataAdminBundle/compare/42.3.1...42.x`.
```
Comment (core23) [inline project/CONTRIBUTING.md]: Maybe use something like `AcmeBundle`?

Comment (kunicmarko20) [reply project/CONTRIBUTING.md]: That won't give you the actual UI

Comment (core23) [reply project/CONTRIBUTING.md]: Maybe the sandbox?

Comment (OskarStark) [reply project/CONTRIBUTING.md]: The sandbox is not well maintained

**Pull request merged**